creme/billing/templates/billing/bricks/base/lines.html

```diff
@@ -38,7 +38,7 @@
                 <div class="bline-hidden-fields" style="display: none">{{form.user}} {{form.DELETE}} {{form.cremeentity_ptr}}</div>
 
                 <div class="bline-fields restorable_{{form.instance.id}}">
-                    {% include 'billing/bricks/frags/line-fields.html' %}
+                    {% include line_edit_form_template %}
```

Comment:
Mark this as a breaking change in CHANGELOG in the "billing" section (eg: the template 'templates/billing/bricks/base/lines.html' needs the variable "line_edit_form_template" to be filled).